bundles/org.openhab.ui/web/build/webpack.config.js

```diff
@@ -20,7 +20,7 @@ const buildSourceMaps = process.env.SOURCE_MAPS || false
 const isCordova = target === 'cordova'
 
 // const apiBaseUrl = 'http://openhab:8080' // 'http://demo.openhab.org:8080' // 'http://localhost:8080'
-const apiBaseUrl = 'http://localhost:8080'
+const apiBaseUrl = 'http://192.168.1.251:8080'
```

Comment:
Is this temporary?

Reply:
That's me doing the very thing I warned against in the contributing.md file ;)
Thanks!

Reply:
was wondering if we should not have something like a webpack.config.local.js file that gets assigned/imported into the main webpack file to support user overides?  I have two  different changes that i am maintaining here.

Reply:
Could be valuable indeed if we can make this work.